lib/util.py

```diff
@@ -542,55 +542,62 @@ def format_time(timestamp):
 # Takes a timestamp and returns a string with the approximation of the age
 def age(from_date, since_date = None, target_tz=None, include_seconds=False):
     if from_date is None:
-        return "Unknown"
+        return _("Unknown")
 
     from_date = datetime.fromtimestamp(from_date)
     if since_date is None:
         since_date = datetime.now(target_tz)
+    else:
+        if isinstance(since_date, int) or isinstance(since_date, float):
```

Comment:
Tiny nit: you can put `isinstance(since_date (int, float))` and Python knows to check if `isinstance` of any 1 item in that tuple.

Reply:
Otherwise.. this great.  Thanks for simplifying the API!!

Reply:
Ah, great, I'll update that.